docs/netdata-agent/backup-and-restore-an-agent.md

```diff
@@ -1,5 +1,11 @@
 # Backing up a Netdata Agent
 
+> **Note**
+>
+> **Note**
+> 
+> Users are responsible for backing up, recovering, and ensuring their data's availability because Netdata stores data locally on each system due to its decentralized architecture.
```

Comment:
```suggestion
> **Note**
> 
> Users are responsible for backing up, recovering, and ensuring their data's availability because Netdata stores data locally on each system due to its decentralized architecture.
```